tools/sgsqlc/tools.go

```diff
@@ -0,0 +1,28 @@
+package sgsqlc
+
+import (
+	"context"
+	"os/exec"
+
+	"go.einride.tech/sage/sg"
+	"go.einride.tech/sage/sgtool"
+)
+
+const version = "v1.12.0"
+
+// nolint: gochecknoglobals
+var commandPath string
+
+func Command(ctx context.Context, args ...string) *exec.Cmd {
+	sg.Deps(ctx, PrepareCommand)
+	return sg.Command(ctx, commandPath, args...)
+}
+
+func PrepareCommand(ctx context.Context) error {
+	binary, err := sgtool.GoInstall(ctx, "github.com/kyleconroy/sqlc/cmd/sqlc", version)
```

Comment:
Let's use the prebuilt binary instead? https://github.com/kyleconroy/sqlc/releases - usually faster download, especially on GitHub Actions

Reply:
Done!